code/modules/client/loadout/loadout_hat.dm

```diff
@@ -36,7 +36,6 @@
 	role_replacements = list(
 		"Captain" = /obj/item/clothing/head/beret/captain,
 		"Head of Personnel" = /obj/item/clothing/head/beret/hop,
-		"SolGov Representative" = /obj/item/clothing/head/beret/solgov,
```

Comment:
this whole file is actually a WS file, should delete it 😏

Reply:
Gone